crates/node-maintainer/src/lockfile.rs

```diff
@@ -31,6 +47,26 @@ impl Lockfile {
         &self.packages
     }
 
+    pub fn to_json(&self) -> JsonDocument {
+        let mut doc = json!({});
+        doc["name"] = json!(self.root.name.as_ref());
+        doc["version"] = json!(self.root.version);
+        doc["lockfileVersion"] = json!(self.version);
```

Comment:
you want to use _NPM_'s version, not ours

Reply:
Where can I get the version info? I have to hardcode a `2`, I guess

Reply:
you have to hardcode it based on what format you're actually generating, yes.